staging/src/k8s.io/apiserver/pkg/endpoints/filters/audit.go

```diff
@@ -173,6 +179,8 @@ type auditResponseWriter struct {
 	event *auditinternal.Event
 	once  sync.Once
 	sink  audit.Sink
+	// omitStages are skipped, i.e. not sent to the backend. But their information is still recorded in the context Event for later stages.
```

Comment:
> omitStages are skipped

Is this comment necessary? `once` and `sink` are not set to the backend also :) Basically, the only explanation why `omitStages` field is there is `b/c it's used later` and this description is not very informative. Maybe remove the comment altogether?

Reply:
Good for me.
Deleted.